src/agent/test_query_pipeline.py

```diff
@@ -20,7 +20,7 @@
 load_dotenv()
 
 Settings.llm = Ollama(
-    model="llama3.1:8b-instruct-q4_0",
+    model=os.getenv("OLLAMA_MODEL_NAME", "lllama3.1:8b-instruct-q4_0"),
```

Comment:
there's a typo here - extra `l` in `lllama3.1:8b-instruct-q4_0`

Reply:
Sorry for the typo, it snuck in when I was doing a global find and replace